src/renderer/components/settings/field/file.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+    <label>
+        {{ field.display || field.name }}
+        <input
+            type="file"
+            :name="field.name"
+            :minLength="field.minLength"

```

Comment:
no `minLength` and `maxLength` to a file input

Reply:
Fixed [here #77](https://github.com/FirstLegoLeague/Launcher/pull/77)